src/llscan.cc

```diff
@@ -78,9 +94,39 @@ bool FindObjectsCmd::DoExecute(SBDebugger d, char** cmd,
 
   result.Printf(" ---------- ---------- \n");
   result.Printf(" %10" PRId64 " %10" PRId64 " \n", total_objects, total_size);
+}
 
-  result.SetStatus(eReturnStatusSuccessFinishResult);
-  return true;
+
+void FindObjectsCmd::DetailedOutput(SBCommandReturnObject& result) {
+  std::vector<DetailedTypeRecord*> sorted_by_count;
+  for (auto it : llscan.GetDetailedMapsToInstances()) {
+    sorted_by_count.push_back(it.second);
+  }
+
+  std::sort(sorted_by_count.begin(), sorted_by_count.end(),
+            TypeRecord::CompareInstanceCounts);
+  uint64_t total_objects = 0;
+  uint64_t total_size = 0;
+
+  result.Printf(
+      "   Repr. Obj.  Instances  Total Size  Properties  Array Size Name\n");
```

Comment:
The name `Repr. Obj.` does not seem very intuitive here. Maybe `Sample`? Also the name `Elements` seems to be more accurate than `Array Size` since we are talking about the number of properties & elements here.